lib/gprd.gi

```diff
@@ -560,7 +560,7 @@ local gc,hc,S,info;
 	    homomorphisms:=[gh,hh],
 	    projections:=[Projection(S,1)*InverseGeneralMapping(gc),
 			  Projection(S,2)*InverseGeneralMapping(hc)]);
-  S:=Group(GeneratorsOfGroup(S));
+  S:=Group(GeneratorsOfGroup(S),One(S));
```

Comment:
Do we have a test case for those semidirect product fixes?

Reply:
I did not prepare one (as this line is anyhow run through by the tests, and I did not have test cases for all of the situations potentially caught).